api/internal/localizer/builtinplugins.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localizer
+
+import "sigs.k8s.io/kustomize/kyaml/yaml"
+
+// localizeBuiltinGenerators localizes built-in generators with file paths.
+// Note that this excludes helm, which needs a repo.
+type localizeBuiltinGenerators struct {
```

Comment:
nit: I suggest adding`var _ kio.Filter = &localizeBuiltinGenerators{}` and `var _ kio.Filter = &localizeBuiltinTransformers{}` somewhere near their type definitions so that the intention of implementing the Filter interface is obvious when looking at this file (and it helps developers leverage the compiler/IDE more by immediately showing an error here if someone ever tries to tweak it)